sdk/js/src/utils/consts.ts

```diff
@@ -316,8 +317,9 @@ const TESTNET = {
     nft_bridge: undefined,
   },
   aptos: {
-    core: undefined,
-    token_bridge: undefined,
+    core: "0x25de93a587d5dc2d7e673663554b7e1d5b00de5d1d38341a896a2141bba5c5c9",
```

Comment:
let's not put the mainnet stuff in until it's actually deployed

Reply:
@kcsongor this was your change, taking it out